Sources/SwiftProtobuf/FieldTag.swift

```diff
@@ -0,0 +1,65 @@
+// SwiftProtobuf/Sources/SwiftProtobuf/FieldTag.swift - Describes a binary field tag
```

Comment:
This kind of goes back to the whole `protobufData:` vs. `data:` debate. Binary encoding *is* the fundamental representation of protobufs—the text and JSON encodings are extras. Adding "Binary" or some other qualifier to these just adds noise without much other benefit. It should also be noted that the other language implementations use similar terms unqualified, and it's probably better not to diverge from those so that it's easier for users of one library to map concepts across easily.